packages/xo-web/src/common/intl/messages.js

```diff
@@ -2301,6 +2301,7 @@ const messages = {
   displayAuditRecord: 'Display record',
   noAuditRecordAvailable: 'No audit record available',
   refreshAuditRecordsList: 'Refresh records list',
+  auditLogsRecordDisabled: 'The record of the logs is disabled',
```

Comment:
@pdonias Any idea for a better message?

Reply:
`'User actions are not saved'` to match the plugin option's description.

Reply:
`The record of the user actions is inactive`?

Reply:
`User actions recording is currently inactive`